root/scripts/talker/mechanic.txt

```diff
@@ -5056,6 +5069,44 @@ Rule EllisStoryReac01GoatMechanic
 	Response EllisStoryReac01GoatMechanic
 }
 
+Response C13M2EllisStoryMechanic
```

Comment:
Can do a generic story rule with all stories instead, need Kerry's help first via a new criteria to filter out chapter 1 maps.

Reply:
We _could_ do that, but I decided to use stories that had some context to the situation